internal/webapi/server.go

```diff
@@ -109,6 +109,11 @@ func (a *Api) Authenticate(auth authentication.Authenticator[APIAuthCredientals]
 	// Check if credientals are correct
 	token, err := auth.CreateToken(packet)
 
+	if err == authentication.InvalidCredientalsError || err == authentication.NotAuthenticatedError {
+		slog.Info("HELLLLLLLLLLLLLLLO")
+		return &femto.Response[types.Unit]{Status: http.StatusUnauthorized}, nil
```

Comment:
Could you add a test for this behaviour?

Reply:
I'm pretty sure there's already a test for this - put the test was using GET instead of POST so was passing. updating